### DISCUSSION TITLE
crashed gollum by mistake

### DISCUSSION THREAD

**Discussion by qwertz44** [gollum/gollum#1931] [Q&A]
hi,
i´ve used the whole domain including path to the gollum instance as name for the renaming operation and since that gollum isn´t working properly. Many pages and folders are now missing. It this a issue to be reported under "Issues"?

It also throws a exception when trying to open "Latest changes":
```
NoMethodError at /latest_changes
undefined method `[]' for nil:NilClass
file: commit.rb location: stats line: 235
```

**Comment by benjaminwil** [comment]
Hi @cf7urb0,

I have tried to reproduce your issue, but I'm not sure I have all of the information I need to reproduce from your post. Have you learned anything since you posted this?

If you're able to provide some form of reproduction steps, please feel free to submit an issue for this.

**Reply by qwertz44** [reply]
Hi @benjaminwil ,
thanks for answering. I will try to give you more detailed steps for reproducing this issue:
1. create folders "A" & "B"
2. create a file in folder "A"
3. rename this file to "https://yourgollum.wiki/B"

**Reply by bartkamphorst** [reply]
RE point 1: create one folder named `"A" & "B"`? Or two folders, one named `A` and one named `B`?

**Reply by qwertz44** [reply]
@bartkamphorst sorry i meant two folders
